sno-setup.sh

```diff
@@ -168,7 +168,7 @@ sudo virt-install \
   --connect 'qemu:///system' \
   -n ${hostname} \
   --vcpus 8 \
-  --memory 32768 \
+  --memory 24576 \
```

Comment:
IIRC we used that value in the CI due https://issues.redhat.com/browse/OCPBUGS-8036, so now probably it could be lowered even to 16G

Reply:
OK, fixed this now and brought it down to 16G!